### DISCUSSION TITLE
Method Group to Expression

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#5177] [Q&A]
Is it possible to convert a method group containing only one item to an expression without using the lambda syntax?

ie:

```
public void DoSomething(Expression E){ }
...

DoSomething(MyMethod);
```

**Comment by 333fred** [comment]
No, it is not. Do you want an Expression that invokes that method, or do you want the body of that method as the Expression? Either way, the only thing we support is lambda to Expression.
